apps/src/p5lab/spritelab/constants.js

```diff
@@ -100,6 +100,9 @@ export const exampleSprites = {
 
 // Big numbers in some blocks can cause performance issues. Combined with live-preview,
 // this results in hanging the tab and students unable to edit their blocks. We
-// guard against this by capping numbers where needed.
+// guard against this by capping number of sprites.
+// The user will receive a workspace alert at SPRITE_LIMIT_WARNING. But the actual cap
+// is MAX_NUM_SPRITES
 export const MAX_NUM_SPRITES = 1000;
+export const SPRITE_LIMIT_WARNING = MAX_NUM_SPRITES - 1;
```

Comment:
NIT: When I see "SPRITE_LIMIT_WARNING" I tend to think the variable contains a string. Something such as "SPRITE_LIMIT_THRESHOLD" might be a clearer indicator that the variable contains a number.

Reply:
Great idea - thanks! I'll use SPRITE_WARNING_THRESHOLD.